project-base/storefront/components/Blocks/Product/ProductsSlider.tsx

```diff
@@ -4,9 +4,12 @@ import { mediaQueries } from 'components/Theme/mediaQueries';
 import { ListedProductFragmentApi } from 'graphql/generated';
 import { useGtmSliderProductListViewEvent } from 'hooks/gtm/productList/useGtmSliderProductListViewEvent';
 import { useKeenSlider } from 'keen-slider/react';
-import { useState } from 'react';
+import React, { useState } from 'react';
```

Comment:
This (adding `React`) should not be needed.

Reply:
Thanks, wondering how it got there, and why it was no reported as unused 🤔 
fixed in f452f8e295